api-mobile/src/paths/context/internal/{target}.ts

```diff
@@ -0,0 +1,158 @@
+'use strict';
+
+import { RequestHandler, Response } from 'express';
+import { Operation } from 'express-openapi';
+import { ALL_ROLES } from '../../../constants/misc';
+import { getLogger } from '../../../utils/logger';
+import { getDBConnection } from '../../../database/db';
+
+const defaultLog = getLogger('activity');
+
+export const GET: Operation = [getContext()];
+
+GET.apiDoc = {
+  description: 'Fetch internal contextual data for a single point.',
+  tags: ['activity','RISO','IPMA'],
+  security: [
+    {
+      Bearer: ALL_ROLES
+    }
+  ],
+  parameters: [
+    {
+      in: 'query',
+      name: 'lon',
+      required: true
+    },
+    {
+      in: 'query',
+      name: 'lat',
+      required: true
+    },
+    {
+      in: 'path',
+      name: 'target',
+      required: true
+    }
+  ],
+  responses: {
+    200: {
+      description: 'Target value',
+      content: {
+        'application/json': {
+          schema: {
+            type: 'object',
+            properties: {
+              target: {
+                type: 'string'
+              }
+            }
+          }
+        }
+      }
+    },
+    401: {
+      $ref: '#/components/responses/401'
+    },
+    503: {
+      $ref: '#/components/responses/503'
+    },
+    default: {
+      $ref: '#/components/responses/default'
+    }
+  }
+};
+
+
+/**
+ * ## getPlanningArea
+ * Get the Invasive Plant Management Area
+ * @param lon {float} Longitude
+ * @param lat {float} Latitude
+ * @param res {object} Express response object
+ * @param attr {string} The postgres table attribute to target
+ * @param table {string} The postgres table to target
+ */
+const getPlanningArea = async (
+  lon: any,
+  lat: any,
+  res: Response,
+  attr: string,
+  table: string
+) => {
+  const connection = await getDBConnection();
+
+  if (!connection) {
+    throw {
+      status: 503,
+      message: 'Failed to establish database connection'
+    };
+  }
+
+  const sql = `
+    select
+      target.${attr} "target"
+    from
+      public.${table} "target"
+    where
+      public.st_intersects(
+        public.geography(
+          public.st_setSrid(
+            public.st_point(${lon},${lat})
+          ,4326)
+        ),target.geog
+      )
+  `;
+
+  try {
+    const response = await connection.query(sql);
```

Comment:
You bet. I can make the change in my next branch.

Reply:
Would be best to account for any changes we think are appropriate now, rather than later.  Otherwise they are more likely to get missed, and just become tech debt.